controllers/v1/userProjects.js

```diff
@@ -1125,5 +1126,33 @@ module.exports = class UserProjects extends Abstract {
                 }
         })
     }
+    async userProjects(req) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                if (req.query.stats) {
+                    //convert req.query.stats string to Boolean value
+                    req.query.stats = UTILS.convertStringToBoolean(req.query.stats);
+                }
+
+                let listOfCreatedProjects = await userProjectsHelper.listUserProjects({
```

Comment:
from here you can call userprojectoverview function @borkarsaish65

Reply:
done that now